security/certificate_types_descriptions/olm-certificates.adoc

```diff
@@ -13,3 +13,17 @@ All certificates for Operator Lifecycle Manager (OLM) components (`olm-operator`
 When installing Operators that include webhooks or API services in their `ClusterServiceVersion` (CSV) object, OLM creates and rotates the certificates for these resources. Certificates for resources in the `openshift-operator-lifecycle-manager` namespace are managed by OLM.
 
 OLM will not update the certificates of Operators that it manages in proxy environments. These certificates must be managed by the user using the subscription config.
+
+[role="_additional-resources"]
+.Next steps
```

Comment:
I am tempted to make the same comment about next steps sections [needing an ID](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), but I feel like that rule only applies to assembly-level next steps, and not for a module. Just wanted to point it out

Reply:
You are right. I forgot to add id's for the assembly-level sections. The rule of thumb I usually use (though I forgot while writing :sweat_smile:) is that labels don't have an ID and headings do.